cli/template/base/tsconfig.json

```diff
@@ -27,7 +27,8 @@
     "**/*.ts",
     "**/*.tsx",
     "**/*.cjs",
-    "**/*.mjs"
+    "**/*.mjs",
+    "../extras"
```

Comment:
what is this for?

Reply:
eehem... prob something that snuck in from an autoimport somewhere... will remove!